components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/Events/PendingAlarmsOptionsModel.cs

```diff
@@ -49,6 +49,18 @@ public class PendingAlarmsOptionsModel {
                 TimeSpan.FromMilliseconds(SnapshotInterval.Value) : (TimeSpan?)null;
         }
 
+        /// <summary>
+        /// ConditionIdIndex
```

Comment:
I know the code looks like this... but can we make these summaries better? A sentence that describes the property.